code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -218,15 +218,15 @@
 		mode = SYRINGE_DRAW
 		update_icon()
 
-/obj/item/chems/syringe/proc/handleBodyBag(var/obj/structure/closet/body_bag/bag, var/mob/living/carbon/user)
+/obj/item/chems/syringe/proc/handleBodyBag(var/obj/structure/closet/body_bag/bag, var/mob/living/user)
```

Comment:
noting that we should probably `snake_case` these procs at some point, along with dbcore typepaths and such. not a request for changes